src/demo-app/accessibility/accessibility.html

```diff
@@ -0,0 +1,36 @@
+<md-sidenav-container class="demo-root" fullscreen>
+  <md-sidenav #start>
+    <md-nav-list>
+      <a *ngFor="let navItem of navItems"
+         md-list-item
+         (click)="start.close()"
+         [routerLink]="[navItem.route]">
+        {{navItem.name}}
+      </a>
+    </md-nav-list>
+    <button md-button (click)="start.close()">CLOSE</button>
+  </md-sidenav>
+  <div>
+    <md-toolbar color="primary">
```

Comment:
I would omit this toolbar and everything in it. These controls are useful for development but are just noise for a11y testing. 

Instead of the toolbar we should just have a single `<h1>` with the title of the specific demo.